### PR TITLE
README updated - docker steps added

### PR DESCRIPTION
**Pull request by ruzickap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,3 +126,74 @@ To do that, simply run:
...
+
+```bash
+apt update
+apt install -y curl git golang jq unzip
+```
+
+Install Terraform:
+
+```bash
+TERRAFORM_LATEST_VERSION=$(curl -s https://checkpoint-api.hashicorp.com/v1/check/terraform | jq -r -M '.current_version')
```
Comment (trawler) [inline README.md]: please note that the terraform build target is pinned to version 0.12.9, and it is fetched to the system via 'make build'

**Review by trawler — CHANGES_REQUESTED** [review]
I would prefer that the manual steps below be added as a make build target in the Makefile.
IE - `make build` to build via docker, and `make local` to build a local target.

File: README.md (hunk 1 above)
Comment (ruzickap) [reply README.md]: Fixed....

**Comment by ruzickap** [comment]
I can add the Dockerfile later.
For now I will just keep the details also in README file.

**Comment by ruzickap** [comment]
Closing - content of this PR was moved to #28.

**Pull request closed without merging**